internal/cmd/kafka/command_link_create.go

```diff
@@ -266,5 +389,22 @@ func (c *linkCommand) getRemoteClusterMetadata(cmd *cobra.Command, linkMode link
 			return "", "", err
 		}
 		return remoteClusterId, bootstrapServer, nil
+	} else if linkMode == Bidirectional {
```

Comment:
`switch` now that there's more than two types?

Reply:
Agreed. Changed to `switch` here and a few other places.